docs/supported-sources/zendesk.md

```diff
@@ -0,0 +1,84 @@
+# Zendesk
+
+[Zendesk](https://www.zendesk.com/) is a cloud-based customer service and support platform. It offers a range of features including ticket management, self-service options, knowledgebase management, live chat, customer analytics, and conversations.
+
+ingestr supports Zendesk as a source.
+
+The Zendesk supports two authentication methods:
+
+- OAuth Token
+- API Token
+
+All resources supports API Token except "chats resources" i.e Zendesk Chat API.
+
+## URI Format
+
+The URI format for Zendesk is as follows:
+
+```plaintext
+zendesk://<sub-domain>?oauth_token=<oauth_token>
+```
+
+```plaintext
+zendesk://<sub-domain>?api_token=<api_token>&email=<email>
+```
+
+URI parameters:
+
+- `subdomain`: Unique zendesk subdomain that can be found in account url
```

Comment:
could you please give an example URI somewhere here to see what the subdomain would look like?